### ISSUE TITLE
Unable to add a commuter into a trip - Organisation Dashboard 

### ISSUE THREAD

**Issue by thelulaloop** [LULA-LOOP/.github#25]
Add a title
Unable to add a commuter into a trip
Current Behaviour 
When the admin tried to add a booking into a specific trip it gave an error message instead of adding the commuter to the trip on the Smp5 organization dashboard.
Expected Behaviour 
The booking supposed to be added into a trip.
Steps to Reproduce
1. looged into  Amp5 organization dashboard
2. Go to trips, on today's 7PM trips look for the Amplify5 : Southern Suburbs (Ottery/Wynberg/GP & Surroundings) - ORANGE
3. Clicked on managed bookings - tried to add a commuter( Black Europa)
4. Observe
Reproduction Link
https://organizations.lulaloop.co.za/trips/222150
Additional Information

What platform is this issue being experienced on?
Organisation Dashboard
Bug submitted by: @Larissa Mawonde

**Comment by thelulaloop** [comment]
@ArseneMwenze please have a look before the team does?

**Comment by ArseneMwenze** [comment]
Hi @thelulaloop 

The functionality does work on my side. However, here are the dependencies that might be confusing for the users :

1. For a trip to appear under the "trips" page on the Org dashboard, the trip needs to be assigned first from the operator dashboard.

2. For a trip to be assigned from the operator dashboard, the schedule needs to have reached the cut off time.

3. If a schedule has already reached the cut off time, the "add commuter" functionality will not work and the org dashboard will display the message "Cut off time for loop schedule has passed and hence cannot make booking."

In other words, there is no bug on the "add commuter" functionality. But in order to "add commuters" on a schedule, the trip must display on the "trips page" in the org dash. But for a trip to display on the "trips page", it needs to be assigned. But if it's assigned it means it reached cut off time. But if it's reached cut off time then the "add commuter" button won't work anyways. The functionality is therefore "unusable" unless you play around/bypass the cut off time blocker each time from the org dash.
